src/client/proxyless/client-function-executor.ts

```diff
@@ -0,0 +1,45 @@
+import {
+    createReplicator,
+    FunctionTransform,
+    ClientFunctionNodeTransform
+} from './replicator';
+import evalFunction from './eval-function';
+import Replicator from 'replicator';
+import { ExecuteClientFunctionCommandBase } from '../../test-run/commands/observation';
+
+export default class ClientFunctionExecutor {
+    public readonly fn: Function;
+    public readonly replicator: Replicator;
+    public readonly dependencies: unknown;
+    public readonly command: ExecuteClientFunctionCommandBase
+
+    public constructor (command: ExecuteClientFunctionCommandBase) {
+        this.command      = command;
+        this.replicator   = this._createReplicator();
+        this.dependencies = this.replicator.decode(this.command.dependencies);
+
+        this.fn = evalFunction(this.command.fnCode, this.dependencies);
+    }
+
+    public getResult (): Promise<unknown> {
+        // eslint-disable-next-line hammerhead/use-hh-promise
```

Comment:
To remove `eslint` comment, I propose to use the hammerhead's Promise instead of native Promise since there is no differences between them in this context.

Reply:
Browsers, that support a remote protocol, must support the native Promise. Now, the proxyless bundle includes the hammerhead's Promise, but I'll improve the bundle and remove unnecessary pieces later. We need to create a separate `.eslintrc` for the proxyless bundle to resolve this issue.

Reply:
Ok. When let it be as is.